cpp-package/tests/ci_test.sh

```diff
@@ -36,6 +36,9 @@ cp ../../build/cpp-package/example/lenet_with_mxdataiter .
 cp ../../build/cpp-package/example/resnet .
 ./resnet 5
 
+cp ../../build/cpp-package/example/resnet .
+./inception-bn 5
+
 cp ../../build/cpp-package/example/mlp .
```

Comment:
should we also add the mlp_csv example here?

Reply:
@stu1130 `ccsgd` was deprecated long time ago, actually, I will update to change `ccsgd` in all other examples.